ui/src/message_popup/author/new_question_page.jsx

```diff
@@ -0,0 +1,79 @@
+/* eslint no-console: "off" */
+import React from 'react';
+import _ from 'lodash';
+
+import QuestionPage from './question_page.jsx';
+import * as Api from '../../helpers/api.js';
+
+export default React.createClass({
+  displayName: 'NewQuestionPage',
+
+  propTypes: {
+    loaded: React.PropTypes.bool,
+    allQuestions: React.PropTypes.object,
+  },
+
+  getDefaultProps(){
```

Comment:
Same suggestion here about removing the props, and testing in a way that will exercise the queries to the server.